item-spec/item-spec.md

```diff
@@ -215,6 +216,105 @@ Deprecation notice: GeoTiff previously used the media type `image/vnd.stac.geoti
 Cloud Optimized GeoTiffs used `image/vnd.stac.geotiff; profile=cloud-optimized`.
 Both can still appear in old catalogues, but are deprecated and should be replaced.
 
+#### Item Properties for Assets
+
+As detailed above, Items contain properties, which are the main source of metadata for searching across Items. Many content extensions can add further property fields as well. Any property that can be specified for an Item can also be specified for a specific asset. This can be used to override a property defined in the Item, or to specify fields for which there is no single value for all assets.
+
+**It is important to note that the STAC API does not faciliate searching across Asset properties in this way, and this should be used sparingly.** It is primarily used to define properties at the Asset level that may be used during use of the data instead of for searching. 
```

Comment:
Maybe also add some words about how repetitive or "larger" chunks can be put in asset definition using the extension?

Reply:
I was going to tackle the idea of moving out parts of the assets into the Item Asset definition as a separate PR

Reply:
Okay, fine for me if there's a tracking issue?!

Reply:
Not yet - will create one